src/pymodaq_plugins_avantes/hardware/AvaSpec_ULS2048CL_EVO_Controller.py

```diff
@@ -0,0 +1,447 @@
+import time,sys
+import numpy as np
+from datetime import datetime
+from pymodaq_plugins_avantes.hardware import avaspec
+
+AvsDeviceType =         {
```

Comment:
better to use enums. Like this you can use autocmopletion in your code using those entries